osworld_setup/s3/bbon/generate_facts.py

```
@@ -90,7 +90,7 @@ def extract_step_num(filename):
         try:
             with open(trajectory_file, "r") as f:
                 trajectory_lines = f.readlines()
-        except:
+        except Exception:
             pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)